Revit_Core_Engine/Modify/SortColumEndpoints.cs

```diff
@@ -25,34 +25,33 @@
 using BH.oM.Geometry;
 using System.ComponentModel;
 
-namespace BH.Engine.Adapters.Revit
+namespace BH.Revit.Engine.Core
 {
     public static partial class Modify
     {
         /***************************************************/
-        /****              Public methods               ****/
+        /****               Public Methods              ****/
         /***************************************************/
 
-        [Description("Flip the input line if its start point is higher than its endpoint.")]
-        [Input("line", "Line to check.")]
-        [Output("line", "The original line if its start point is already below its endpoint. Otherwise, the line flipped.")]
-        public static Line SortEndpoint(this Line locationLine)
+        [Description("Flip the input line of a BHoM Column object if its start point is higher than its endpoint. This allows pushing it to Revit")]
+        [Input("line", "Column line to check.")]
+        [Output("line", "The original column line if its start point is already below its endpoint. Otherwise, the same line flipped.")]
+        public static Line SortColumEndpoints(this Line columnLine)
         {
-            if (locationLine.Start.Z == locationLine.End.Z)
+            if (columnLine.Start.Z == columnLine.End.Z)
```

Comment:
We should always include tolerance in geometrical queries to avoid numerical noise and negligible inaccuracies. So if you want to compare 2 geometrical objects for equality of any type, it is worth using `BH.oM.Geometry.Tolerance.Distance` (or others from this class):
```suggestion
            if (Math.Abs(columnLine.Start.Z - columnLine.End.Z) <= BH.oM.Geometry.Tolerance.Distance)
```